frontend/src/hooks/workspace/useUrlWorkspacename.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import {useCallback, useEffect, useState} from "react";
+import {WorkspaceService} from "../../services/WorkspaceService";
+import {Workspace} from "../../fi/hg/dashboard/types/Workspace";
+import {VoidCallback} from "../../fi/hg/core/interfaces/callbacks";
+import {LogService} from "../../fi/hg/core/LogService";
+
+const LOG = LogService.createLogger('useUrlWorkspaceName');
+
+export function useUrlWorkspaceName (
```

Comment:
There should be documentation what this hook does ... and the name probably should also hint that.